src/bitcoin-wallet.cpp

```diff
@@ -24,6 +24,7 @@
 #include <string>
 #include <tuple>
 
+static constexpr int CONTINUE_EXECUTION=-1;
```

Comment:
```suggestion
static constexpr int CONTINUE_EXECUTION{-1};
```

forgot to run clang-format?

Reply:
Thanks, done.